stories/RangeInput.stories.js

```diff
@@ -14,4 +17,28 @@ storiesOf('RangeInput', module)
   }))
   .add('with min and max', () => ({
     template: `<ais-range-input attribute="price" :min=10 :max=50></ais-range-input>`,
+  }))
+  .add('using the default slot', () => ({
+    template: `
+      <ais-range-input attribute="price">
+        <template slot-scope="{refine, currentRefinements}">
+          <form  @submit.prevent="refine(min, max)" >
+            <label>
+              <input type="number" :max="this.max"  :placeholder="this.max" :value="currentRefinements && currentRefinements[0]" @change="min = $event.currentTarget.value"/>
```

Comment:
no big deal at all, but if it gets over 80 lines I personally move the props to their own lines.

Reply:
I was wondering about that